org.eclipse.xtext.xtext.wizard/src/org/eclipse/xtext/xtext/wizard/ParentProjectDescriptor.xtend

```diff
@@ -457,6 +457,17 @@ class ParentProjectDescriptor extends ProjectDescriptor {
 						</repository>
```

Comment:
whaout about other places e.g. line 98, 450, 473

Reply:
The wizard tests never use snapshots so expectations didn't need an update. If that's what you meant.